### PR TITLE
Add aptosconnect

### PR DESCRIPTION
**Pull request by 0xmaayan**

### CODE REVIEW

File: packages/wallet-adapter-core/src/WalletCore.ts [hunk 1]
```diff
@@ -173,6 +177,25 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
     });
   }
 
+  private fetchSDKWallets = () => {
+    const aptosStandardWallets: AptosStandardWallet[] = [];
+
+    SDKWallets.map((sdkWallet: any) => {
```
Comment (hardsetting) [inline packages/wallet-adapter-core/src/WalletCore.ts:183]: can we use a strong type here?

File: packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { DevTWallet } from "@atomrigslab/aptos-wallet-adapter";
+import { AptosConnectWalletPlugin } from "@aptos-connect/wallet-adapter-plugin";
+import { Network } from "@aptos-labs/ts-sdk";
+
+const isProd = process.env.NODE_ENV === "production";
+
+let aptosNetwork = isProd ? Network.MAINNET : Network.TESTNET;
+
+const wallets: any = [new DevTWallet()];
```
Comment (hardsetting) [inline packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts:9]: Why are we using `any` here?

File: packages/wallet-adapter-core/src/WalletCore.ts [hunk 3]
```diff
@@ -173,6 +177,25 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
...
+      // // initiate the SDK wallet
+      // const sdkWallet = new wallet();
```
Comment (hardsetting) [inline packages/wallet-adapter-core/src/WalletCore.ts:185]: remove?

File: packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+if (typeof window !== "undefined") {
+  wallets.push(new AptosConnectWalletPlugin({ network: aptosNetwork }));
```
Comment (hardsetting) [inline packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts:12]: `AptosConnectWalletPlugin` is the plugin for the wallet adapter v1.
You should use `AptosConnetWallet` for the new wallet standard interface

File: packages/wallet-adapter-core/src/WalletCore.ts [hunk 5]
```diff
@@ -173,6 +177,25 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
...
+
+      const isValid = isWalletWithRequiredFeatureSet(sdkWallet);
+      // TODO add user opt-in check
+      if (isValid) {
+        const standardWallet = sdkWallet as AptosStandardWallet;
+        standardWallet.readyState = WalletReadyState.Installed;
+        aptosStandardWallets.push(standardWallet);
+        this.standardizeStandardWalletToPluginWalletType(standardWallet);
+      }
```
Comment (hardsetting) [inline packages/wallet-adapter-core/src/WalletCore.ts:194]: A little confused here.
Why do we have to cast, and manually set `readyState`?

**Comment by 0xmaayan** [comment]
closing in favor of https://github.com/aptos-labs/aptos-wallet-adapter/pull/297

**Pull request closed without merging**